src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -104,7 +104,8 @@ const useSaveAnonChatHook = () => {
       console.log(e);
     } finally {
       refresh('channelList');
-      refresh('chat');
+      // refresh('chat');
+      refreshWithId('chat', initAnonymousChat.message.cid);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of the general `refresh('chat')` with the more specific `refreshWithId('chat', initAnonymousChat.message.cid)` is a positive change. This should lead to more efficient data handling as it only refreshes the chat with the specified ID instead of all chats.
```javascript
// refresh('chat');
refreshWithId('chat', initAnonymousChat.message.cid);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->